sklearn/cluster/tests/test_hdbscan.py

```diff
@@ -304,7 +304,7 @@ def test_hdbscan_centers(algorithm):
     accurate to the data.
     """
     centers = [(0.0, 0.0), (3.0, 3.0)]
-    H, _ = make_blobs(n_samples=1000, random_state=0, centers=centers, cluster_std=0.5)
+    H, _ = make_blobs(n_samples=2000, random_state=0, centers=centers, cluster_std=0.5)
```

Comment:
oh wow, I thought this is more like, increasing from 10 samples to 100. But with 1000 samples and two dimensions, this should be quite stable, having it so unstable is concerning. I understand why this is though, but still.

cc @Micky774

Reply:
Right, this seems to be the most brittle piece. I'm not sure why, but it should really only be used for illustrative purposes anyways (with exceptions if you _really know_ the shape of your data). Even then it still is too fickle for full comfort. I'd love to investigate in detail if I get the time for it 😭